clusterclaims/apply.sh

```diff
@@ -322,7 +322,10 @@ while [[ ("$CC_PEND_CONDITION" != "False" || "$CD_HIB_CONDITION" != "False" || "
     oc get clusterclaim ${CLUSTERCLAIM_NAME} -n ${CLUSTERPOOL_TARGET_NAMESPACE} -o json > $CC_JSON
     CC_NS=`jq -r '.spec.namespace' $CC_JSON`
     if [[ "$CC_NS" != "null" ]]; then
-        oc get clusterdeployment $CC_NS -n $CC_NS -o json > $CD_JSON
+        oc get clusterdeployment $CC_NS -n $CC_NS -o json > $CD_JSON 2> $CD_ERROR
```

Comment:
Shoot. It doesn't like this:
```
./apply.sh: line 305: $CD_ERROR: ambiguous redirect
./apply.sh: line 325: $CD_ERROR: ambiguous redirect
```

Reply:
I made a poor assumption that this was storing it to a variable. I've created an error file instead.